plugins/emacs/emacs.plugin.zsh

```diff
@@ -10,23 +10,22 @@
 # - Configuration changes made at runtime are applied to all frames.
 
 
-if "$ZSH/tools/require_tool.sh" emacs 23 2>/dev/null ; then
+if "$ZSH/tools/require_tool.sh" emacs 24 2>/dev/null ; then
     export EMACS_PLUGIN_LAUNCHER="$ZSH/plugins/emacs/emacsclient.sh"
 
     # set EDITOR if not already defined.
     export EDITOR="${EDITOR:-${EMACS_PLUGIN_LAUNCHER}}"
 
     alias emacs="$EMACS_PLUGIN_LAUNCHER --no-wait"
     alias e=emacs
+    # open terminal emacsclient
+    alias te="$EMACS_PLUGIN_LAUNCHER -nw"
 
     # same than M-x eval but from outside Emacs.
     alias eeval="$EMACS_PLUGIN_LAUNCHER --eval"
     # create a new X frame
     alias eframe='emacsclient --alternate-editor "" --create-frame'
 
-    # to code all night long
```

Comment:
if you're trying to encourage usage of [thefuck](https://github.com/nvbn/thefuck) instead
![i approve](https://cloud.githubusercontent.com/assets/860471/16606043/64e9c7d0-42e9-11e6-86c7-6dbac259a44d.gif)